.lando.yml

```diff
@@ -0,0 +1,64 @@
+name: simplytest-me
+recipe: drupal9
+
+config:
+  php: "7.3"
+  webroot: web
+  xdebug: false
+  database: mariadb:10.3
+
+events:
+  # Clear cache after a database import
+  post-db-import:
+    - appserver: cd $LANDO_WEBROOT && drush cache-rebuild --yes
+
+services:
+  appserver:
+    run:
+      - cd $LANDO_MOUNT && composer install
+
+  nodejs:
+    type: node:10
+    globals:
+      gulp-cli: latest
+    run:
+      - cd $LANDO_MOUNT && npm install
+      - cd $LANDO_MOUNT && npm install gulp
+      - cd $LANDO_MOUNT && gulp build
```

Comment:
This will not work because `npm` will look for `package.json` in the root of the repository. It is in the theme directory. Compare with PR #7.